Podfile.lock

```diff
@@ -18,11 +18,6 @@ PODS:
   - DoubleConversion (1.1.5)
   - Down (0.6.6)
   - Folly (2018.10.22.00):
-    - boost-for-react-native
-    - DoubleConversion
-    - Folly/Default (= 2018.10.22.00)
-    - glog
-  - Folly/Default (2018.10.22.00):
```

Comment:
These changes came from a `rake dependencies`
Doing the same on `develop` generates the same diff.

There might have been a bad merge at some point

Reply:
CI is failing and seems related to this 🤔

Reply:
Removing `Pods` folder did the trick.